### PR TITLE
doc: use sphinx to generate documentation

### PR DESCRIPTION
**Pull request by kan-fu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,39 @@
...
 
+- Click the Profile link (top right corner) to access your account profile.
 
-## License
+- Access the Web Services API tab and click Copy Token.
 
+- If you forget your token, you can always find it in your Oceans 3.0 account profile.
 
----
+### How to get started with Oceans 3.0 API?
```
Comment (kan-fu) [inline README.md:34]: I added the two paragraphs. Might need some improvements here.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,39 @@
...
 
+There are some [tutorials](https://oceannetworkscanada.github.io/api-python-client/Tutorial/index.html) to get you started.
 
+We suggest to start with how Oceans 3.0 API works (key concepts like discovery, data product delivery, real-time and archive files),
```
Comment (kan-fu) [inline README.md:38]: I added this paragraph. Might need some improvements here.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,39 @@
...
+and then check the documentation for the client library in Matlab or [Python](https://oceannetworkscanada.github.io/api-python-client/index.html).
```
Comment (kan-fu) [inline README.md]: I added the two paragraphs here. Might need some improvements here.

File: README.md (hunk 2 above)
Comment (aschlesin) [inline README.md]: We suggest to start with how Oceans 3.0 API works (key concepts like: data and instrument discovery, data product delivery, real-time data access and archived files),

**Review by aschlesin — APPROVED** [review]
This looks pretty solid and good to me. Thanks!

**Pull request merged**